client/driver/fabfile.py

```diff
@@ -132,6 +144,25 @@ def get_result():
     local(cmd)
 
 
+def _ready_to_start_controller():
+    file_path = '/home/shulij/ottertune/client/driver/oltp.log'
+    if not os.path.exists(file_path):
+        return False
+    else:
+        lines = open(file_path).readlines()
+        return len(lines) >= 15 and len(lines[14].split(' ')) > 2
```

Comment:
I don't think check a specific line is a good way to determine the starting time. Try to find the content in the log. Like 'staring the experiment. '